game.js

```diff
@@ -0,0 +1,359 @@
+/*
+ #    #     #     ####   #    #    ##    #    #   #####  #    #  #####   #####
+ ##   #     #    #       #    #   #  #   ##   #     #    #    #  #    #  #    #
+ # #  #     #     ####   ######  #    #  # #  #     #    #    #  #    #  #    #
+ #  # #     #         #  #    #  ######  #  # #     #    # ## #  #####   #####
+ #   ##     #    #    #  #    #  #    #  #   ##     #    ##  ##  #   #   #
+ #    #     #     ####   #    #  #    #  #    #     #    #    #  #    #  #
+
+*/
+rebtn = document.querySelector('#rebtn');
+rediv = document.querySelector('.rebtn');
+rebtn.onclick = function() {
+    location.reload();
+}
+document.querySelector('#onebtn').onclick = function() {
+    document.querySelector('.oneopt').classList.remove('hide');
+    document.querySelector('.oneopta').classList.remove('hide');
+    document.querySelector('.twobtn').classList.add('hide');
+    document.querySelector('.onebtn').classList.add('hide');
+}
+var convert = new Map();
```

Comment:
I'd suggest you to use JS objects for a cleaner code, which is much easier to understand. For other developers to use Ctrl+F to find what exactly goes in `bmap1` is really stupid.